skimage/future/graph/rag.py

```diff
@@ -293,6 +299,17 @@ def edges_iter(self, *args, **kwargs):
         except AttributeError:
             return super(RAG, self).edges(*args, **kwargs)
 
+    def subgraph(self, nbunch):
+        if self.version1:
+            return super(RAG, self).subgraph(nbunch)
+        else:
+            G = super(RAG, self).subgraph(nbunch)
+            G.node = G._node
```

Comment:
```G.node``` should probably be set to ```G.nodes``` instead of ```G._node``` when making a v2.0 graph look like a v1.x graph.